setup.py

```diff
@@ -1,13 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
 from setuptools import setup
 
 setup(
     name="omero-cli-zarr",
-    version="0.0.1",
+    version="0.1.0.dev0",
     packages=["omero_zarr", "omero.plugins"],
     package_dir={"": "src"},
     description="Plugin for exporting images in zarr format.",
-    url="",
-    license="",
-    author="",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Environment :: Plugins",
+        "Intended Audience :: Developers",
+        "Intended Audience :: End Users/Desktop",
+        "Intended Audience :: Information Technology",
+        "License :: OSI Approved :: GNU General Public License v2 " "or later (GPLv2+)",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    author="The Open Microscopy Team",
     author_email="",
+    license="",
```

Comment:
I was going to suggest something here, but ended up in a rabbit hole... https://github.com/pypa/packaging.python.org/issues/95

Reply:
Agreed. https://packaging.python.org/guides/distributing-packages-using-setuptools/#license suggests that the `classifiers` should be the reference place for the project license in all cases. Happy to remove the `license` keyword if the vocabulary if unclear as I could not find a list of official terms either.

Reply:
https://github.com/ome/omero-py/blob/dab7f1790ec0c0b5f0db0beaf3948b328a41e469/setup.py#L179-L233 doesn't have a `license` field and no-one's said anything.